### PR TITLE
Wording enhancement

### LINKED ISSUE #16004: Typo?
It says:
&gt; "For example, an authorization policy can specify by name which authorization scheme (or schemes) should be used to authenticate the user."

Shouldn't it be "[..]which authentication scheme [..] to authenticate the user" ? If it is not a mistake - i.e. if I am just failing to understand it - then it would be helpful to add an explanatory sentence in there to prevent this sort of confusion for future readers. Thx. :-)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 393b3379-6061-50a4-0975-b3d1d82b660b
* Version Independent ID: 48e4e204-ea22-c225-0cbc-0d811bdb90ed
* Content: [Overview of ASP.NET Core Authentication](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/?view=aspnetcore-3.1#feedback)
* Content Source: [aspnetcore/security/authentication/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/authentication/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @mjrousos
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #16004

### CODE REVIEW

File: aspnetcore/security/authentication/index.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ An authentication scheme is a name which corresponds to:
 * An authentication handler.
 * Options for configuring that specific instance of the handler.
 
-Schemes are useful as a mechanism for referring to the authentication, challenge, and forbid behaviors of the associated handler. For example, an authorization policy can specify by name which authorization scheme (or schemes) should be used to authenticate the user. When configuring authentication, it's common to specify the default authentication scheme. The default scheme is used unless a resource requests a specific scheme. It's also possible to:
+Schemes are useful as a mechanism for referring to the authentication, challenge, and forbid behaviors of the associated handler. For example, a named authorization policy can specify which authorization scheme (or schemes) should be used to authenticate the user. When configuring authentication, it's common to specify the default authentication scheme. The default scheme is used unless a resource requests a specific scheme. It's also possible to:
```
Comment (mjrousos) [inline aspnetcore/security/authentication/index.md]:
```suggestion
Schemes are useful as a mechanism for referring to the authentication, challenge, and forbid behaviors of the associated handler. For example, an authorization policy can use scheme names to specify which authentication scheme (or schemes) should be used to authenticate the user. When configuring authentication, it's common to specify the default authentication scheme. The default scheme is used unless a resource requests a specific scheme. It's also possible to:
```

**Comment by guardrex** [comment]
Should that be “a scheme name” (singular)?

**Comment by guardrex** [comment]
Nevermind ... guess not given the following parenthetical remark

**Comment by mjrousos** [comment]
Maybe. Authorization policies can specify a scheme name or (if they accept multiple authentication methods) multiple scheme names indicating which authentication schemes should be used. It's a little awkward to word.

**Pull request merged**